packages/yoroi-extension/features/step_definitions/support-steps.js

```diff
@@ -0,0 +1,117 @@
+// @flow
+
+import { When, Then } from 'cucumber';
+import { expect } from 'chai';
+import {
+  descriptionTextArea,
+  emailInput,
+  supportButton,
+  getPlatformLocator,
+  frameTitle,
+  successText,
+  submitButton,
+  platformSelector,
+  acceptCheckbox,
+} from '../pages/supportPage';
+import { By } from 'selenium-webdriver';
+
+const axios = require('axios');
+
+const mailsacAPIKey = 'k_a5fgBik5hsHx2DjoYXGFy8U8dhs3d8XIUzHXt';
```

Comment:
It should be removed from here. The key will be added to Github Secrets keys and after that you can get it from the keys and put into env